package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Restaurant-App",
+  "version": "1.0.0",
+  "description": "Restaurant App built with Javascript, using the Webpack library.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Your README references `build` script but you don't actually have build script which uses your webpack commands

Reply:
Scripts like `build`, `start`, and `development` are required according to this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from pathright.